src/protocols/secure_channel/CASESession.h

```diff
@@ -249,10 +272,10 @@ class DLL_EXPORT CASESession : public Messaging::ExchangeDelegate, public Pairin
     Transport::FabricTable * mFabricsTable = nullptr;
     Transport::FabricInfo * mFabricInfo    = nullptr;
 
-    struct SigmaErrorMsg
-    {
-        SigmaErrorType error;
-    };
+    uint8_t mResumptionId[kCASEResumptionIDSize];
+    uint8_t mInitiatorRandom[kSigmaParamRandomNumberSize];
```

Comment:
The reason for maintenance of mInitiatorRandom in a member should be stated ->

`// Sigma1 initiator random, maintained to be reused post-Sigma1, such as when generating Sigma2 S2RK key`